src/main/java/com/kodilla/ecommercee/domain/dto/CartDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+public class CartDto {
+
+    private Long Id;
+    private List<CartItemDto> cart;
```

Comment:
I would like to add here:
 private Long userId;

it could be helpful but it's only proposition.


Marcin, We got 13 commits and a lot of confusion but you did a great job

Reply:
Thanks

Reply:
You probably can pass information with @RequestParam. In that case you will not need userId. I'm not sure if one way is right way to do and other is not, or both are or both are not.